common/src/main/java/us/myles/ViaVersion/protocols/protocol1_11to1_10/data/PotionColorMapping.java

```diff
@@ -0,0 +1,64 @@
+package us.myles.ViaVersion.protocols.protocol1_11to1_10.data;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+import us.myles.ViaVersion.api.Pair;
+
+public class PotionColorMapping {
+
+    //<oldData> to <newData, isInstant> mapping
+    private static final Int2ObjectMap<Pair<Integer, Boolean>> POTIONS = new Int2ObjectOpenHashMap<>(17, 1.0F);
```

Comment:
Replace the 17 with 37 (the number of entries it will have); this pre-sizes the map to that specific size